### PR TITLE
[WIP] Change download modal and general index style

### PR DESCRIPTION
**Pull request by Soitora**
I can't seem to get how to trigger another modal (SweetAlert2) within an existing one, so this is why it's WIP.

### CODE REVIEW

**Comment by arkon** [comment]
> I can't seem to get how to trigger another modal

What're you trying to do?

File: src/.vuepress/styles/palette.styl [hunk 1]
```diff
@@ -23,5 +23,5 @@ $MQNarrow = 959px
 $MQMobile = 767px
 $MQMobileNarrow = 419px
 // Fonts
-$buttonFontFamily = 'Open Sans', Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif
+$fontFamily = Metropolis, 'Open Sans', Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif
```
[the remark this answers is not in the document]
Comment (Soitora) [reply src/.vuepress/styles/palette.styl:26]: Was experimenting and seeing your reaction to it

**Comment by Soitora** [comment]
> > I can't seem to get how to trigger another modal
> 
> What're you trying to do?

Shit you are fast this time...

I'm trying to get the `preview` link (`<a>`) inside the footer to trigger another modal, kind of like how pressing the `Download` button triggers a secondary modal. Honestly can't figure it out as having a `@click=showDownloadPreview` or something within the HTML just renders it *literally*, with no real function.

I got locally the jQuery script working for doing at least the Analytics info and window assign, but it didn't recognize SweetAlert2 there, seemingly it's not recognized as a global function or something

**Comment by arkon** [comment]
But why trigger another model? I feel like a direct download link is fine.

**Comment by Soitora** [comment]
It's fine if it don't trigger it, but at least I'd want to close the current modal, but if you're fine with it not reacting to that specific click then by all means I can work with that

**Comment by Soitora** [comment]
Abandoned since I have rewrote it on a new branch.

**Pull request closed without merging**